Frontend/src/pages/Voting-Overview.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from "@chakra-ui/react"
+import React, { useContext, useEffect, useState } from "react"
+import StateContext from "../state/stateContext"
+
+function VotingOverview() {
+  const { daoContract } = useContext(StateContext)
+
+  const [currentWinningProposal, setCurrentWinningProposal] = useState("")
+
```

Comment:
Maybe add some comment here about what you try to accomplish here? Some TODO?

Reply:
Yep makes sense. Done